types/react-redux/tsconfig.json

```diff
@@ -22,4 +22,4 @@
         "noEmit": true,
         "forceConsistentCasingInFileNames": true
     }
-}
```

Comment:
I noticed some types were off in the test file and originally changed `strictNullChecks` to true, but I didn't know if there was a backstory to why this was set to false. I think `strictNullChecks` should always be set to true, otherwise TS is perfectly happy with the following:

```ts
function getLength(value: string): number {
  return value.length;
}

getLength(null); // TS is happy with `strictNullChecks=false`, but this will result in a runtime TypeError
```

Reply:
I'm not sure if there's a backstory either but I definitely agree with you on this one.

Reply:
same